client/settings/transactions/index.js

```diff
@@ -80,6 +84,9 @@ const Transactions = () => {
 					<span className="input-help-text" aria-hidden="true">
 						{ `${ accountStatementDescriptor.length } / ${ ACCOUNT_STATEMENT_MAX_LENGTH }` }
 					</span>
+
+					<SupportEmailInput onErrorMessage={ setSaveDisabled } />
+					<SupportPhoneInput onErrorMessage={ setSaveDisabled } />
```

Comment:
The state of both inputs needs to be stored separately in order to determine whether saving is disabled or not. Right now:

1. I opened the page with valid values in both fields.
2. Deleted the email ➡️  The save button got disabled.
3. Deleted the phone number ➡️ The save button remain disabled.
4. Started typing an email (which is not validated) ➡️ The save button immediately got enabled, even though the phone number remains empty.

Reply:
Another great catch! Fixed in this commit https://github.com/Automattic/woocommerce-payments/pull/5471/commits/a6a3b894ac75bdb5c9a76b5c36fcd97862d27c52

I've also updated Card Reader settings page too as it has the same issue. https://github.com/Automattic/woocommerce-payments/pull/5471/commits/404fde21f38f129b3db6226a442168809e38020a